### PR TITLE
Implement crawler.

### PR DESCRIPTION
**Pull request by asiday**
Changes from previous pull request:
- Move test DB to the test/resources folder
- Delete not used method/imports/classes

### CODE REVIEW

File: crawler/src/main/java/dev/asida/crawler/dao/ArticleRepository.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * create your own queries using certain naming convention.
+ */
+public interface ArticleRepository extends JpaRepository<Article, Integer> {
+    /**
+     * Create your "dynamic finder" using certain naming convention.
+     * Here it looks for a Author name, that contains parameter "string".
+     * @param string is a letter or part of a string that is a part of the Author name.
+     * @return A list of Articles, that contains this "string" in Author name.
+     */
+    List<Article> findAllByAuthorContaining(String string);
```
Comment (yambay) [inline crawler/src/main/java/dev/asida/crawler/dao/ArticleRepository.java:24]: Do we need this method?

**Review by yambay — CHANGES_REQUESTED** [review]

File: crawler/src/main/java/dev/asida/crawler/dao/ArticleRepository.java (hunk 1 above)
Comment (asiday) [reply crawler/src/main/java/dev/asida/crawler/dao/ArticleRepository.java:24]: No, we do not need this method. It was created only for educational purposes.

Comment (asiday) [reply crawler/src/main/java/dev/asida/crawler/dao/ArticleRepository.java:24]: Deleted.

File: crawler/src/main/java/dev/asida/crawler/json/Article.java [hunk 2]
```diff
@@ -1,50 +1,153 @@
...
-    private LocalDate publishedAt;
+    private String urlToImage;
+    private LocalDateTime publishedAt;
+    private String content;
+
+    // Default constructor is required for JPA
+    public Article() {
+    }
+
+    public Article(Integer id, String author, String title, String description,
```
[the remark this answers is not in the document]
Comment (asiday) [reply crawler/src/main/java/dev/asida/crawler/json/Article.java:33]: Both constructors are deleted. use setters to create a proper object.

**Review by yambay — APPROVED** [review]

**Pull request merged**